sdk/search/azure-search-documents/azure/search/documents/_service/_index.py

```diff
@@ -0,0 +1,249 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from .edm import Collection, ComplexType
+from ._generated.models import Field
+
+if TYPE_CHECKING:
+    from typing import Any, Dict, List
+
+__all__ = ("ComplexField", "SearchableField", "SimpleField")
+
+
+def SimpleField(
+    name,
+    type,  # pylint:disable=redefined-builtin
+    key=False,
```

Comment:
Obviously we can change these later - but would like most of these being keyword only

Reply:
They should be keyword only arguments, yes. There is no logical ordering between them as far as I can see.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/11188